library/Zend/Stdlib/ArrayUtils.php

```diff
@@ -238,12 +238,7 @@ public static function iteratorToArray($iterator, $recursive = true)
                 continue;
             }
 
-            if ($value instanceof Traversable) {
```

Comment:
The `instanceof` check is in place because it is less expensive than the `is_array()` call.

It looks like duplication, but this is really performance-sensitive code :-(